q2_cutadapt/_demux.py

```diff
@@ -282,15 +291,24 @@ def demux_paired(seqs: MultiplexedPairedEndBarcodeInSequenceDirFmt,
     ):
         raise ValueError("'forward_cut' and 'reverse_cut' need to be set to "
                          "the same number when using the 'mixed_orientation' "
-                         "mode")
+                         "mode.")
+
+    if (
+        mixed_orientation
+        and anchor_forward_barcode != anchor_reverse_barcode
+    ):
```

Comment:
What do you think about the case where `anchor_reverse_barcode` is true but `reverse_barcodes` is none? Seems like raises an error could be better than a user falsely assuming that a reverse barcode trimming is happening. Commenting on this section because it's a similar check conceptually.

Reply:
That's right. I added a check to make sure `reverse_barcodes` is not `None` if `anchor_reverse_barcode` is given